docs/mobile-apps/mdm-support.md

```diff
@@ -36,15 +34,28 @@ Setting up App VPNs is made easy, enabling you to connect to your internal firew
 **Retained App Data**
 All apps can be retained on the devices of your choice, and the Apple ID used for App Store login can also be retained on the device.
 
-## Obtaining the BundleID
+## Installing MDM
 
-To effectively leverage the MDM support and enforce policies any iOS application (iOS15 and above), you need to obtain their BundleIDs. Follow these steps during live testing:
+1. Start any Live Testing session with the private device where you want to install your MDM.
+2. Sign in with your AppleID and download Intune or install your MDM using a specific link.
+3. Login with your service account to your MDM/Intune portal.
+4. After successful login, download and install the MDM profile through settings.
+5. Navigate to Settings/VPN and Device Management, click on Management Profile, and install the MDM.
+6. Once installed, click on the Home Screen.
+7. Confirm that all the apps related to your company-managed profile have been successfully downloaded.
 
-1. Open a livetesting session with an iOS Device.
-2. Open Device logs with VERBOSE mode.
+## Retrieving the BundleID for allowlisting your apps
+
+To prevent our cleaning to remove your application after every session, this guide will help you to obtain the bundleID for these applications. 
```

Comment:
```suggestion
To prevent our cleaning from removing your application after every session, this guide will help you to obtain the bundleID for these applications. 
```